### ISSUE TITLE
Connecting to Skydemon

### ISSUE THREAD

**Issue by rolakon** [astenlund/fs2ff#17]
Hello, ;)
Thank you for this app connecting Skydemon to FS2020. This is very helpful.
I run FS2020 (FS) and Skydemon (SD) on my little LAN. Link is established without a problem. BUT:
when I choose to see other traffic, my aircraft is shown as traffic as well. So SD is continously alerting about traffic in 5 'o clock same altitude. Is there any parameter or other chance to avoid seeing my aircraft as traffic?
![fs2ff_quest](https://user-images.githubusercontent.com/70313262/97105301-0fd0c380-16ba-11eb-9f67-18114dac64e9.png)

**Comment by astenlund** [comment]
Huh, I have not seen this. Will investigate, thank you.

**Comment by rolakon** [comment]
Looking forward :) :)

**Comment by rolakon** [comment]
Hello,
are you any near to a solution? 
Best regards Roland

**Comment by c997aru** [comment]
Sorry, I haven't had time to look into this yet. Hopefully I'll get to it sometime this week. /A

**Comment by rolakon** [comment]
No problem... Just take your time... Everthing is fine... 👍

**Comment by jakobsack** [comment]
Hi @astenlund , I stumbled across the same issue when testing your great tool together with SkyDemon. For whatever reason your own plane is broadcasted as traffic. I "solved" the issue by checking `if (DataTrafficEnabled && tfk.TailNumber == "ASXGS")` in MainViewModel.cs .
The question is how to go forward from here. So far I can see two options:

1. Allow the user to enter call signs to ignore for traffic in some textbox
2. Do some magic by comparing the last position received with the position of the traffic. If the difference of these values is negligible then ignore the traffic.

What's your opinion on this? Which option would you prefer? Do you have more ideas?

**Comment by astenlund** [comment]
@rolakon: sorry for not getting back to you, I realize this is a frustrating bug. I just haven't had much time and energy to spend on this project lately. When I started, I was on an extended furlough, but now I'm back to working full-time again.

Hi @jakobsack. I've been thinking a little about this, and your option 2 is pretty much what I also came up with. Option 1 would be too imprecise.

If you would like to submit a pull-request, I promise to have a look at it! :)

/Andreas
